boa_interner/src/sym.rs

```diff
@@ -193,5 +196,6 @@ create_static_strings! {
     "false",
     "async",
     "of",
-    "target"
+    "target",
+    "__proto__"
```

Comment:
Can this include a comma at the end of line? Would be nice to avoid having to modify the last line every time a new string is added.

Reply:
Unfortunately not as this is a macro call. There may be some way to modify the macro to allow it, but I'm not sure about that.

Reply:
Just change the production of the macro to `( $( $s:literal ),+$(,)? )`